### PR TITLE
Track flash messages in analytics

### PR DESCRIPTION
**Pull request by fofr**
Keep track of the success, error and warning messages displayed to
users by using events.

* Track the alert type as the action
* Track the alert message as the label
* The page URL is captured as the event category automatically.

This will allow tracking of edition saves that happen without ajax.

### CODE REVIEW

**Comment by fatbusinessman** [comment]
data-module="auto-track-event"

What is this sorcery?

**Comment by fofr** [comment]
The AutoTrackEvent module is within the admin template, and has mostly been used by the Transition app.

See: https://github.com/alphagov/govuk_admin_template/blob/master/app/assets/javascripts/govuk-admin-template/modules/auto_track_event.js

**Comment by fatbusinessman** [comment]
Thanks for the explanation, @fofr. :+1:
